tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/resources/template/emailWithIcsMissingVEVENT.eml.mustache

```diff
@@ -0,0 +1,34 @@
+To: {{receiver.name}} <{{receiver.email}}>
+From: {{sender.name}} <{{sender.email}}>
+Subject: Event Invitation from {{sender.name}}
+Message-ID: {{mimeMessageId}}
+Date: Tue, 5 Sep 2017 09:54:16 +0200
+MIME-Version: 1.0
+Content-Type: multipart/mixed;
+ boundary="------------D24E361990BDBA143D4D8794"
+Content-Language: en-US
+
+This is a multi-part message in MIME format.
+--------------D24E361990BDBA143D4D8794
+Content-Type: text/plain; charset=utf-8; format=flowed
+Content-Transfer-Encoding: 7bit
+
+The message has a text attachment.
+
+--------------D24E361990BDBA143D4D8794
+Content-Type: text/calendar; charset=UTF-8; method=REQUEST
+Content-Disposition: attachment;
+Content-Transfer-Encoding: base64
+
+QkVHSU46VkNBTEVOREFSClZFUlNJT046Mi4wClBST0RJRDotLy9TYWJyZS8vU2FicmUgVk9iamVjdCA0LjEuMy8vRU4KQ0FMU0NBTEU6R1JFR09SSUFOCk1F
+VEhPRDpSRVFVRVNUCkJFR0lOOlZUSU1FWk9ORQpUWklEOkFzaWEvSG9fQ2hpX01pbmgKQkVHSU46U1RBTkRBUkQKVFpPRkZTRVRGUk9NOiswNzAwClRaT0ZG
+U0VUVE86KzA3MDAKVFpOQU1FOklDVApEVFNUQVJUOjE5NzAwMTAxVDAwMDAwMApFTkQ6U1RBTkRBUkQKRU5EOlZUSU1FWk9ORQpFTkQ6VkNBTEVOREFS
```

Comment:
What is this? Why not including the `missing_vevent.ics.mustache` file instead like in other mustache eml files?